### ISSUE TITLE
Some report tab content fails to be read out by screen reader when a DataTable is the first child of the first section

### ISSUE THREAD

**Issue by aj-he** [archesproject/arches-her#1309]
When using a screen reader and activating a tab in a report, if the first section has a Datatable element as the first child component, it may fail to read out the content of that tabcontent panel.

On a Mac, it does read the full content out the first time the tab is activated, but if it is deactivated and then activated again it no longer reads out the tab content.
